objects/src/accounts/mod.rs

```diff
@@ -129,6 +129,11 @@ impl Account {
         self.nonce
     }
 
+    /// Returns nonce for this account.
+    pub fn set_nonce(&mut self, nonce: Felt) {
+        self.nonce = nonce;
+    }
```

Comment:
If this is needed only for mock data, should it be gated by `mock` feature? (or if this is needed for testing only, we should gate it with a `testing` flag, or something like that).

Also, might be good to add a `debug_assert` to make sure we don't decrement the nonce.